src/com/google/javascript/jscomp/CompilationLevel.java

```diff
@@ -163,7 +163,7 @@ private static void applyFullCompilationOptions(CompilerOptions options) {
     options.setRemoveAbstractMethods(true);
     options.setRemoveSuperMethods(true);
     options.setReserveRawExports(true);
-    options.setRenamingPolicy(VariableRenamingPolicy.ALL, PropertyRenamingPolicy.ALL_UNQUOTED);
+    options.setRenamingPolicy(VariableRenamingPolicy.ALL, PropertyRenamingPolicy.OFF);
```

Comment:
I know it must have been discussed many times to add a flag for this. It breaks React's plugin system. I didn't have time yet to take a closer look as to what the exact cause was. It is likely easy to fix on the React side.